src/components/form/components/form-group/form-group.stories.js

```diff
@@ -0,0 +1,38 @@
+import { html } from "lit-html";
+import "../../../form";
+import "../../../form-grid";
+import "../../../input-field";
+import "../../../button";
+import "../../../form-message";
+import styles from "../.././styles.scss";
+import inputFieldStyles from "../../../input-field/styles.scss";
+import formGridStyles from "../../../form-grid/styles.scss";
+import buttonStyles from "../../../button/styles.scss";
+import formMessageStyle from "../../../form-message/styles.scss";
+import { stylesheet, docsIntro } from "../../../../../.storybook/utils.js";
+
+export default {
+  title: "native-elements/vl-form/vl-form-group",
+  controls: { hideNoControlsWarning: true },
```

Comment:
Je kan in je story zien dat je hideWarning niets doet. De controls prop hoort in het `parameters` object als ik het goed voor heb.

Reply:
Fixed.